test/e2e/runner/start.go

```diff
@@ -56,21 +56,9 @@ func Start(ctx context.Context, testnet *e2e.Testnet, p infra.Provider) error {
 			return err
 		}
 		if node.PrometheusProxyPort > 0 {
-			logger.Info("start", "msg",
-				log.NewLazySprintf("Node %v up on http://%s:%v; with Prometheus on http://%s:%v/metrics",
-					node.Name,
-					node.ExternalIP,
-					node.ProxyPort,
-					node.ExternalIP,
-					node.PrometheusProxyPort,
-				),
-			)
+			logger.Info("start", "msg", log.NewLazySprintf("Node %v up on http://127.0.0.1:%v; with Prometheus on http://127.0.0.1:%v/metrics", node.Name, node.RPCProxyPort, node.PrometheusProxyPort))
```

Comment:
The IP address is fixed here, instead of using the variable `node.ExternalIP` as in the original code. The same happens in the `logger.Info` line below.

Reply:
We should revert this change, in my opinion.

Actually, the diff of this file should match the output of the following command:

```
git diff main v0.38.x test/e2e/runner/start.go
```

Reply:
Namely, `node.ProxyPort` renamed to `node.RPCProxyPort` should be the only change here.